src/main/common/filter.c

```diff
@@ -86,6 +86,37 @@ void biquadFilterInit(biquadFilter_t *filter, float filterCutFreq, uint32_t refr
     filter->d1 = filter->d2 = 0;
 }
 
+/* sets up a biquad Filter */
+void biquadFilterNotchInit(biquadFilter_t *filter, float filterCutFreq, uint32_t refreshRate, float Q)
```

Comment:
This looks identical to `biquadFilterInit` except with `Q` as a parameter. If so can you change `biquadFilterInit` to be just a call to this, ie
```
void biquadFilterInit(biquadFilter_t *filter, float filterCutFreq, uint32_t filterCutFreq)
{
    biquadFilterNotchInit(filter, filterCutFreq, filterCutFreq, BIQUAD_Q);
}
```